### PR TITLE
Added dotenv gem to Gemfile

### PR DESCRIPTION
**Pull request by ilanusse**
## Summary
Added dotenv gem to Gemfile as requested.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -90,6 +90,9 @@ end
 group :development, :test do
   gem 'awesome_print'
 
+  # Loads ENV variables from .env file in base folder
+  gem 'dotenv-rails'
```
Comment (hdf1986) [inline Gemfile]: We can use a version with this. According to the gemfile.lock will be 2.2.0 :D

Comment (ilanusse) [inline Gemfile]: I personally like doing that but I wasn't sure if people in general like doing that.

Comment (hdf1986) [inline Gemfile]: Both ways sounds good for me :+1:

Comment (mishuagopian) [inline Gemfile]: Hmm. What I don't like about that is we are not fixing gems to any version..

Comment (ilanusse) [inline Gemfile]: That's why I didn't do it @mishuagopian

File: README.md [hunk 2]
```diff
@@ -142,6 +142,28 @@ Set the following variables in your server.
...
+```
+
+and a `.env` file in the project root that looks like this:
+
+```
+FOO=1
+BAR=2
+```
+
+When you load up your application, `Rails.secrets.foo` will equal `ENV['FOO']`, making your environment variables reachable across your Rails app.
```
Comment (epintos) [inline README.md]: Rails.application.secrets.foo

**Pull request merged**